apps/browser/qml/pages/components/TabGridView.qml

```diff
@@ -15,7 +15,7 @@ import org.nemomobile.configuration 1.0
 import "." as Browser
```

Comment:
I think `org.nemomobile.configuration` and `"."` aren't needed and can be removed.

Reply:
done

Reply:
Great, many thanks. Resolving.